django/db/models/query.py

```diff
@@ -435,11 +435,12 @@ def bulk_create(self, objs, batch_size=None):
         if not objs:
             return objs
         self._for_write = True
-        connection = connections[self.db]
+        db = self.db
```

Comment:
@sjoerdjob let's remove the `db` optimization to keep the patch concise. On the other hand I still think we should use `zip`.